erpnext_com/www/docs/user/manual/en/setting-up/articles/integrating-erpnext-with-biometric-attendance-devices.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Integrating ERPNext With Biometric Attendance Devices
+add_breadcrumbs: 1
+show_sidebar: 0
+
+metatags:
+ description: ERPNext has the provision to mark attendance from Check-in and Check-out logs from biometric. There are several possible methods to integrate your biometric device based on the vendor and the available features of your device.
+ keywords: frappe, erpnext, erpnext attendance, biometric device integration, human resource, auto attendance
+---
+
+<!-- add-breadcrumbs -->
+# Integrating ERPNext With Biometric Attendance Devices
+
+The Attendance Punch Logs from the biometric device is a type of Check-in and Check-out log of an employee.
+
+ERPNext has a provision to store these logs in a document called Employee Checkin.
+
+Attendance can then be marked based on the Employee Checkin records and the shift [Auto Attendance Settings](/docs/user/manual/en/human-resources/shift-management#25-auto-attendance-settings) of the employee by using [Auto Attendance](/docs/user/manual/en/human-resources/auto-attendance)
+
+Hence, integrating your Biometric Device --or any access control system that collects IN/OUT--, can be done by the following steps:
+
+  1. [Setting up Auto Attendance to mark attendance from the Employee Checkin](#1-setting-up-auto-attendance-to-mark-attendance-from-the-employee-checkin)
+  1. [Populating the Biometric Punch Logs into ERPNext's Employee Checkin](#2-populating-the-biometric-punch-logs-into-erpnexts-employee-checkin)
+
+## 1. Setting up Auto Attendance to mark attendance from the Employee Checkin
+
+Before you import employee's Check-in and Check-out logs into your ERPNext system. You would have to set up the Employees and their shifts to be able to generate attendance using the Auto Attendance feature in ERPNext.
+
+Please refer to the following link to setup Auto Attendance: [Setup Auto Attendance](/docs/user/manual/en/human-resources/auto-attendance#steps-to-setup-auto-attendance)
```

Comment:
Please refer to the following link to set up Auto Attendance:

Reply:
even the link label should be Set Up Auto Attendance

Reply:
done and done